frontend/packages/ux-editor/src/data/formItemConfig.ts

```diff
@@ -187,6 +187,7 @@ export const formItemConfigs: FormItemConfigs = {
       id: '',
       itemType: 'COMPONENT',
       type: ComponentType.FileUpload,
+      description: 'test',
```

Comment:
Of course not, good catch! I added the description property now because it is required, but I am not sure why the typecheck didn't catch it earlier. Anyways, I'll change it to an empty string.